lesson_7/mosut_sorin/hw_lesson7_decorators_mosut_sorin1.py

```diff
@@ -0,0 +1,38 @@
+'''
+Given the following function:
+
+def greet(name):
+    return "Greetings {}!".format(name)
+
+Create a decorator called uppercase that will uppercase the result
+
+@uppercase
+def greet(name):
+    return "Greetings {}!".format(name)
+
+print(greet("World"))
+"GREETINGS WORLD!"
+
+
+def greet(name):
+    return "Greetings {}!".format(name)
+
+'''
+
+def uppercase(greet):
+
+    def uppercase_in(*args):
+
+        z = str.upper(greet(*args))
```

Comment:
Is the assignment here really needed?

Reply:
no , the assignement is not really needed , i know that.
it could be done like this 
               return str.upper(greet(*args))
but i have done it like this for practice and i have not change it befor commiting, after that i did not want to remake the commit etc.

Reply:
If it's not needed, it should not be in the code ;)

Reply:
Should i make a commit to fix it ?

Reply:
yes, please

Reply:
Done it for all 3 exercises.
Thnx for review.